targets/ath79-generic

```diff
@@ -123,6 +123,11 @@ device('tp-link-archer-d50-v1', 'tplink_archer-d50-v1', {
 	broken = true, -- 64M ath9k + ath10k & power LED not working
 })
 
+device('tp-link-cpe210-v1', 'tplink_cpe210-v1', {
+	manifest_aliases = {
+		'tp-link-cpe210-v1.1',
```

Comment:
Please also add back the `tp-link-cpe210-v1.0` manifest alias, so upgrades from that revision work. It is the same hardware as the v1.1 and has always used the same images.

Reply:
I have no such device at hand and cannot verify it does work. Still?

Reply:
Yes, people will install the image on the device anyways, as it is called just "tp-link-cpe210-v1". So it makes sense to allow upgrades as well.

Reply:
resolved in 3c34a01c64e5c2c3f818e757f8657c945bcd4f58